resources/functionMetadata.php

```diff
@@ -1240,6 +1240,7 @@
 	'join' => ['hasSideEffects' => false],
 	'json_last_error' => ['hasSideEffects' => false],
 	'json_last_error_msg' => ['hasSideEffects' => false],
+	'json_validate' => ['hasSideEffects' => false],
```

Comment:
Please see the top of this file - it's generated and should not be updated manually.